### PR TITLE
project-business-site by Tina

### PR DESCRIPTION
**Pull request by TinaJuren**
Any feedback on the structure and function of the code is appreciated :)

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -3,28 +3,121 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&family=Work+Sans:wght@400;900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+
+    <!-- Hero Section -->
+
+    <header class="hero-section">
+      <video class="hero-video" controls autoplay muted loop>
```
Comment (HIPPIEKICK) [inline code/index.html:17]: Did you try withouth the `controls`? 😇

File: code/index.html [hunk 2]
```diff
@@ -3,28 +3,121 @@
...
+        <source src="./hero-video.mp4" type="video/mp4" media="(max-width: 767px)">
+        <p>This browser doesn't support video...</p>
+      </video>
+      <section class="headline">
+        <h1 class="hero-title">MT Construction</h1>
+        <p class="hero-tagline">building more<br> with less</p>
+      </section>
+      </header>
```
Comment (HIPPIEKICK) [inline code/index.html:25]: This line should not be indented 👀

File: code/index.html [hunk 3]
```diff
@@ -3,28 +3,121 @@
...
-        <p>Lorem ipsum...</p>
-      </div>
-    </section>
-    <!-- dont forget to link the javascript file here! -->
+    <main class="main-area">
+      <section class="contact">
+
+        <form class="contact-form" action="http://httpbin.org/anything" method="POST">
+            
+            <legend class="contact-header">Please contact me</legend>
```
Comment (HIPPIEKICK) [inline code/index.html:34]: Have you seen the `<fieldset>` element? 🤩

File: code/script.js [hunk 4]
```diff
@@ -3,7 +3,52 @@
 // console.log(this) to see how it works
 function toggle() {
   this.classList.toggle("active")
-}
+}
```
Comment (HIPPIEKICK) [inline code/script.js:6]: Add semicolons here as well if you decide to use them

File: code/style.css [hunk 5]
```diff
@@ -1,7 +1,258 @@
+body {
```
Comment (HIPPIEKICK) [inline code/style.css:1]: It all looks especially good on mobile 🤩

Comment (HIPPIEKICK) [reply code/style.css:1]: It looks a little jumpy on desktop though 🤔 
https://user-images.githubusercontent.com/51529201/107059026-9dea3900-67d5-11eb-9ecc-20cdda37b8a1.mov

File: code/style.css [hunk 6]
```diff
@@ -1,7 +1,258 @@
...
+  font-size: 20px;
+  width: 100%;
+  margin-top: 20px;
+}
+
+input:valid {
+  border-style: solid;
+  border-width: 5px;
+  border-color: #3C7A1B;
+}
```
Comment (HIPPIEKICK) [inline code/style.css:113]: Nice!

File: code/style.css [hunk 7]
```diff
@@ -1,7 +1,258 @@
...
+
+input:focus:required:invalid {
+  border-style: solid;
+  border-width: 5px;
+  border-color: #7A3022;
+}
```
Comment (HIPPIEKICK) [inline code/style.css:119]: It might be confusing with the same styling for these

**Review by HIPPIEKICK — APPROVED** [review]
Hi Tina! Check out my comments as well as the [general feedback document](https://www.notion.so/technigo/Week-3-Business-site-9f3ef7b1c0ee480c8a10b9c7e460c475). Happy weekend! :)

**Comment by TinaJuren** [comment]
Thank you so much for your valuable feedback! 
It helped, and will help, me a lot going through your comments and use your tips. It's a few weeks since I did this project, and it's now enjoyable to work with it again. After my pull request, I hardly dared to look at it.

File: code/style.css (hunk 7 above)
Comment (TinaJuren) [reply code/style.css:119]: Yes, I agree. The styling needs to be taken care of :)

File: code/style.css (hunk 6 above)
Comment (TinaJuren) [reply code/style.css:113]: Thanks!

File: code/style.css (hunk 5 above)
Comment (TinaJuren) [reply code/style.css:1]: I will take a look at this. Thank you for the video :)

File: code/index.html (hunk 3 above)
Comment (TinaJuren) [reply code/index.html:34]: Yes, I think I have (here I want to add a suitable emoji but I havn't figured out how yet ;)) I've checked up the fieldset element again, added it and removed the divs I had. It looks cleaner in the code file :) However, I want to got back later and improve the styling. Thanks a lot!

File: code/index.html (hunk 1 above)
Comment (TinaJuren) [reply code/index.html]: Oh, I thought it was needed for the other attributes to work. Now I removed the controls and learned more about the video tag as a result, thank you :)

**Pull request closed without merging**